jupyterlab_git/git.py

```diff
@@ -728,3 +728,53 @@ def _get_tag(self, current_path, commit_sha):
                 )
             )
 
+    def show(self, filename, ref, top_repo_path):
+        """
+        Execute git show<ref:filename> command & return the result.
+        """
+        command = ["git", "show", '{}:{}'.format(ref, filename)]
+        p = subprocess.Popen(
+            command,
+            stdout=PIPE,
+            stderr=PIPE,
+            cwd=top_repo_path
+        )
+        output, error = p.communicate()
+        if p.returncode == 0:
+            return output.decode('utf-8')
+        elif "fatal: Path '{}' exists on disk, but not in '{}'".format(filename, ref) in error.decode('utf-8'):
+            return ""
+        elif "fatal: Path '{}' does not exist (neither on disk nor in the index)".format(filename) in error.decode('utf-8'):
+            return ""
+        else:
+            raise Exception('Error [{}] occurred while executing [{}] command to retrieve plaintext diff.'.format(
+                error.decode('utf-8'),
+                ' '.join(command)
+            ))
+
+    def get_content(self, filename, top_repo_path):
+        """
+        Get the file content of filename.
+        """
+        relative_repo = os.path.relpath(top_repo_path, self.root_dir)
+        model = self.contents_manager.get(path=os.path.join(relative_repo, filename))
+        return model.content
```

Comment:
`contents_manager.get()` returns dict (https://github.com/jupyter/notebook/blob/master/notebook/services/contents/filemanager.py#L417)
```suggestion
        return model['content']
```

Reply:
Thx for testing and the suggestion